packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -115,6 +123,37 @@ class CodeSizeBodyState extends State<CodeSizeBody>
     }
   }
 
+  DropdownButtonHideUnderline _buildDiffTreeTypeDropdown() {
+    return DropdownButtonHideUnderline(
+      child: DropdownButton<DiffTreeType>(
+        value: _diffTreeType,
+        items: [
+          _buildMenuItem(DiffTreeType.combined),
+          _buildMenuItem(DiffTreeType.increaseOnly),
+          _buildMenuItem(DiffTreeType.decreaseOnly),
+        ],
+        onChanged: (newDiffTreeType) {
+          setState(() {
+            _diffTreeType = newDiffTreeType;
+          });
+          controller.loadFakeDiffData(
+            'old_v8.json',
+            'new_v8.json',
+            _diffTreeType,
+          );
+        },
+      ),
+    );
+  }
+
+  DropdownMenuItem _buildMenuItem(DiffTreeType diffTreeType) {
+    // TODO(peterdjlee): Set item's text to the enum's value, not toString().
```

Comment:
see `ProfileGranularityExtension` for an example on how to do this using extension methods

Reply:
Done